razorpay/resources/invoice.py

```diff
@@ -45,3 +45,131 @@ def create(self, data={}, **kwargs):
         """
         url = self.base_url
         return self.post_url(url, data, **kwargs)
+
+    def notify_by(self, invoice_id, medium, **kwargs):
+        """"
+        Send/Resend notifications to customer via email/sms
+
+        Args:
+            invoice_id : Id for trigger notify
+            medium : Medium for triggering notification via email or sms
+
+        Returns:
+            {"success": true}
+        """
+        url = "{}/{}/notify_by/{}".format(self.base_url, invoice_id, medium)
+        return self.post_url(url, {}, **kwargs)
+
+    def cancel(self, invoice_id, **kwargs):
+        """"
+        Cancel an unpaid Invoice with given ID via API
+        It can only be called on an invoice that is not in the paid state.
+
+        Args:
+            invoice_id : Id for cancel the invoice
+        Returns:
+            The response for the API will be the invoice entity, similar to create/update API response, with status attribute's value as cancelled
+        """
+        url = "{}/{}/cancel".format(self.base_url, invoice_id)
+        return self.post_url(url, {}, **kwargs)
+
+    def delete(self, invoice_id, **kwargs):
+        """"
+        Delete an invoice
+        You can delete an invoice which is in the draft state.
+
+        Args:
+            invoice_id : Id for delete the invoice
+        Returns:
+            The response is always be an empty array like this - []
+        """
+        url = "{}/{}".format(self.base_url, invoice_id)
+        return self.delete_url(url, {}, **kwargs)
+
+    def issue(self, invoice_id, **kwargs):
+        """"
+        Issues an invoice in draft state
+
+        Args:
+            invoice_id : Id for delete the invoice
+        Returns:
+            Its response is the invoice entity, similar to create/update API response. Its status now would be issued.
+        """
+        url = "{}/{}/issue".format(self.base_url, invoice_id)
+        return self.post_url(url, {}, **kwargs)
+
+    def edit(self, invoice_id, data={}, **kwargs):
+        """"
+        Update an invoice
+        In draft state all the attributes are allowed.
+
+        Args:
+            invoice_id : Id for delete the invoice
+            data : Dictionary having keys using which invoice have to be updated
+        Returns:
+            {
```

Comment:
Too verbose comment in code, can just say returns json for invoice entity and link to existing docs containing this sample.